content/source/docs/enterprise-beta/registry/using.html.md

```diff
@@ -0,0 +1,88 @@
+sidebar_current: "docs-enterprise2-registry-using"
+---
+
+# Using Modules from the Terraform Enterprise Private Module Registry
+
+~> **Note:** Currently, the private module registry is only enabled on private installs of TFE using Bitbucket Server as the VCS provider. Support for the SaaS version of TFE and other VCS providers is coming in a future update.
+
+By design, the private module registry works much like the [public Terraform Registry](/docs/registry/index.html). If you're already familiar with the public registry, here are the main differences:
+
+- Use TFE's web UI to browse and search for modules.
+- In Terraform configurations, add TFE's hostname as a prefix to the module identifier. For example:
+
+    ```hcl
+    module "vpc" {
+      source = "atlas.hashicorp.com/example_corp/vpc/aws"
+    }
+    ```
+- The second part of the source string (the namespace) is your TFE organization. The module in the example above is loaded from the `example_corp` organization.
+- TFE can automatically access your private modules during Terraform runs. However, if you're running Terraform on the command line, you must configure credentials in your [CLI configuration file (`.terraformrc`)](/docs/commands/cli-config.html).
+
+## Finding Modules
+
+All users in your organization can view your private module registry.
+
+To see which modules are available, click the "Modules" button in TFE's main navigation bar.
+
+![TFE screenshot: Navigation bar with modules button highlighted](./images/using-modules-button.png)
+
+This brings you to the modules page, which lists all available modules.
+
+![TFE screenshot: the list of available modules](./images/using-modules-list.png)
+
+You can browse the complete list, or shrink the list by searching or filtering.
+
+- The "Providers" drop-down filters the list to show only modules for the selected provider.
+- The search field searches by keyword. This only searches the titles of modules, not READMEs or resource details.
+
+### Viewing Module Details and Versions
+
+Click a module's "Details" button to view its details page. Use the "Versions" dropdown in the upper right to switch between the available versions, and use the Readme/Inputs/Outputs/Dependencies/Resources tabs to view detailed documentation and information about a version.
+
+![TFE screenshot: a module details page](./images/publish-module-details.png)
+
+## Using Modules in Terraform Configurations
+
+In Terraform configurations, you can use any private module from your organization's registry. The syntax for referencing private modules in `source` attributes is `<TFE HOSTNAME>/<TFE ORGANIZATION>/<MODULE NAME>/<PROVIDER>`.
+
+```hcl
+module "vpc" {
+  source = "atlas.hashicorp.com/example_corp/vpc/aws"
+}
+```
+
+If you're using the SaaS version of TFE, the hostname is `atlas.hashicorp.com`; private installs have their own hostnames. The second part of the source string (the namespace) is the name of your TFE organization.
```

Comment:
You could also mention that you can leave off the hostname if you are consuming from the public registry.